### PR TITLE
[zh-cn]: create the translation of `navigator.mediaSession` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/mediasession/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Navigator：mediaSession 属性
+slug: Web/API/Navigator/mediaSession
+l10n:
+  sourceCommit: 29d4051fef07473fd0f0eb471528382d6af66e94
+---
+
+{{APIRef("Media Session API")}}
+
+{{domxref("Navigator")}} 接口的只读属性 **`mediaSession`** 返回一个 {{domxref("MediaSession")}} 对象，该对象可用于与浏览器共享元数据和其他有关文档正在处理的媒体的当前播放状态的信息。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/mediasession/index.md:10]:
```suggestion
{{domxref("Navigator")}} 接口的只读属性 **`mediaSession`** 返回一个 {{domxref("MediaSession")}} 对象，该对象可用于与浏览器共享关于文档正在处理的媒体的当前播放状态的元数据和其他信息。
```

File: files/zh-cn/web/api/navigator/mediasession/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+此信息随后可与设备或与操作系统共享，以便为设备的标准媒体控制用户体验提供描述和控制媒体播放的功能。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/mediasession/index.md:12]:
```suggestion
这些信息可以进一步与设备和/或操作系统共享，以便使用设备的标准媒体控制用户体验来描述和控制媒体的播放。
```

File: files/zh-cn/web/api/navigator/mediasession/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+此外，`MediaSession` 接口还提供了 {{domxref("MediaSession.setActionHandler", "setActionHandler()")}} 方法，该方法允许你在用户使用设备控制（例如屏幕上或物理上的播放、暂停、跳转和其他类似控制）时接收事件。例如，互联网广播应用程序可以使用 `setActionHandler()` 来让用户设备上的键盘或其他位置的媒体控制用于控制应用程序的媒体播放。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/mediasession/index.md:14]:
```suggestion
此外，`MediaSession` 接口还提供了 {{domxref("MediaSession.setActionHandler", "setActionHandler()")}} 方法，该方法允许你在用户使用设备控制（例如屏幕上或物理上的播放、暂停、跳转和其他类似控制）时接收事件。例如，网络电台应用程序可以使用 `setActionHandler()` 来让用户设备上的键盘或其他位置的媒体控制用于控制应用程序的媒体播放。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggetions for your reference.

File: files/zh-cn/web/api/navigator/mediasession/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+title: Navigator：mediaSession 属性
+slug: Web/API/Navigator/mediaSession
+l10n:
+  sourceCommit: 29d4051fef07473fd0f0eb471528382d6af66e94
+---
+
+{{APIRef("Media Session API")}}
+
+{{domxref("Navigator")}} 接口的只读属性 **`mediaSession`** 返回一个 {{domxref("MediaSession")}} 对象，该对象可用于与浏览器共享关于文档正在处理的媒体的当前播放状态的元数据和其他信息。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/navigator/mediasession/index.md:10]:
```suggestion
{{domxref("Navigator")}} 接口的 **`mediaSession`** 只读属性返回一个 {{domxref("MediaSession")}} 对象，该对象可用于与浏览器共享关于文档正在处理的媒体的当前播放状态的元数据和其他信息。
```

File: files/zh-cn/web/api/navigator/mediasession/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+这些信息可以进一步与设备和/或操作系统共享，以便使用设备的标准媒体控制用户体验来描述和控制媒体的播放。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/navigator/mediasession/index.md:12]:
```suggestion
这些信息可以进一步与设备及操作系统共享，以便使用设备的标准媒体控制用户体验来描述和控制媒体的播放。
```

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thank you! @T34-active @skyclouds2001

**Pull request merged**